src/index.js

```diff
@@ -220,7 +220,7 @@ const getAuthorizationToken = async (
   return fetch(url.toString(), {
     method: 'POST',
     body: params,
-  }).then((res) => res.json());
+  }).then((res) => (typeof res?.json === 'function' ? res.json() : res));
```

Comment:
@dauden  Should we do this for the other functions as well? eg: revokeAuthorizationToken

Reply:
Yeah! my bad, I will push change now

Reply:
@A-Tokyo I have changed all of the function 

thanks

Reply:
My pleasure, merging now